cpp/src/io/json/reader_impl.cu

```diff
@@ -169,7 +169,7 @@ std::unique_ptr<table> create_json_keys_info_table(
                                          record_starts.data().get(),
                                          record_starts.size(),
                                          key_counter.data(),
-                                         info_table_mdv.data(),
+                                         d_info_table_mdv.data(),
```

Comment:
Just pass it by value?  
```suggestion
                                         *d_info_table_mdv,
```

Reply:
@jrhemstad  It's an optional parameter, so I need to pass an optional-like thing (i.e. pointer). This is the best way I found to do this.